src/current/v23.2/physical-cluster-replication-overview.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Physical Cluster Replication
+summary: Follow a tutorial to set up physical replication between a primary and standby cluster.
+toc: true
+docs_area: manage
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} CockroachDB physical cluster replication continuously sends all data at the byte level from a _primary_ cluster to an independent _standby_ cluster. Existing data and ongoing changes on the active primary cluster, which is serving application data, replicate asynchronously to the passive standby cluster.
+
+In a disaster recovery scenario, you can [_cut over_]({% link {{ page.version.version }}/cutover-replication.md %}) from the unavailable primary cluster to the standby cluster. This will stop the replication stream, reset the standby cluster to a point in time where all ingested data is consistent, and mark the standby as ready to accept application traffic.
+
+{% comment %}A lot of links at the end of this section to general CV docs{% endcomment %}
+
+## Use cases
+
+You can use physical cluster replication in a disaster recovery plan to:
+
+- Meet your RTO (Recovery Time Objective) and RPO (Recovery Point Objective) needs. Physical cluster replication provides lower RTO and RPO than [backup and restore](backup-and-restore-overview.html). {% comment %}link here to upcoming DR tool comparative table{% endcomment %}
+- Automatically replicate everything in your primary cluster to recover quickly from a control plane or full cluster failure.
+- Cover different regions when you cannot use individual [multi-region clusters](multiregion-overview.html). Primary and standby clusters can exist in different regions.
+- Avoid conflicts in data after recovery because the replication completes to a transactionally consistent state as of a certain point in time.
+
+## Features
+
+- **Asynchronous byte-level replication**: When you initiate a replication stream, it will replicate byte-for-byte all existing user data and associated metadata (e.g., zone configurations) in the primary cluster to the standby cluster asynchronously. From then on, it will continuously replicate user data and metadata in the primary to the standby. Physical cluster replication will automatically replicate changes related to operations such as, schema changes, user and privilege modifications, and zone configuration updates without any manual work.
```

Comment:
```suggestion
- **Asynchronous byte-level replication**: When you initiate a replication stream, it will replicate byte-for-byte all existing user data and associated metadata (e.g., zone configurations) in the primary cluster to the standby cluster asynchronously. From then on, it will continuously replicate user data and metadata in the primary to the standby. Physical cluster replication will automatically replicate changes related to operations such as schema changes, user and privilege modifications, and zone configuration updates without any manual work.
```

Reply:
Removed!